repository-data/application/src/main/resources/hcm-config/configuration/translations/types.yaml

```diff
@@ -292,3 +292,18 @@ definitions:
       /en:
         jcr:primaryType: hipposys:resourcebundle
         jcr:name: Details (reusable)
+    /hippo:configuration/hippo:translations/hippo:types/hee:expanderPanel:
+      jcr:primaryType: hipposys:resourcebundles
+      /en:
+        jcr:primaryType: hipposys:resourcebundle
+        jcr:name: Expander Panel
```

Comment:
Lets use Title case like the others for:
Expander panel
Expander group
Expander group (reusable)